scripts/globals/spells/waterga_v.lua

```diff
@@ -0,0 +1,21 @@
+-----------------------------------
+-- Spell: Waterga V
+-----------------------------------
+require("scripts/globals/magic_utils/spell_damage")
+require("scripts/globals/msg")
+-----------------------------------
+local spell_object = {}
+
+spell_object.onMagicCastingCheck = function(caster, target, spell)
+    if caster:isPC() then -- Mob Only Spell.
```

Comment:
this isn't necessary. a player normally can't obtain it to cast it, we do not need to block it. If you choose to GM yourself the spell to get it, its not a bug its just not the projects problem either.